flexget/plugins/plugin_deluge.py

```diff
@@ -793,7 +811,24 @@ def add_entry(entry, opts):
 
                     log.verbose('Adding %s to deluge.' % entry['title'])
                     if magnet:
-                        return client.core.add_torrent_magnet(magnet, opts)
+                        # we need to set paused to false so the magnetization begins immediately
+                        # NOTE: is this the correct config property and value?
+                        opts['add_paused'] = False
+
+                        id = client.core.add_torrent_magnet(magnet, opts)
+
+                        magnetization_timeout = entry.get('magnetization_timeout', config.get('magnetization_timeout'))
+                        if magnetization_timeout > 0:
+                            log.info('Waiting %d seconds for "%s" to magnetize' % (magnetization_timeout, entry['title']))
+                            magnetized = _wait_for_files(id, magnetization_timeout).getResult()
```

Comment:
You should be able to just call this function normally. i.e. drop the `.getResult()`